File: typescript-client-sdk/src/sdk/models/operations/putcrmeventattendee.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class PutCrmEventAttendeeResponseBody extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=accessToken" })
  accessToken?: string;

  @SpeakeasyMetadata({ data: "json, name=eventAttendee" })
  eventAttendee?: shared.EventAttendeeUpdate;
}


export class PutCrmEventAttendeeResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  responseBody?: PutCrmEventAttendeeResponseBody;
}

File: go-client-sdk/pkg/models/operations/putcrmeventattendee.go
package operations

import (
	"github.com/speakeasy-sdks/orb-sdks/go-client-sdk/pkg/models/shared"
)

type PutCrmEventAttendeeResponseBody struct {
	AccessToken   *string                     `json:"accessToken,omitempty"`
	EventAttendee *shared.EventAttendeeUpdate `json:"eventAttendee,omitempty"`
}

type PutCrmEventAttendeeResponse struct {
	ContentType  string
	StatusCode   int64
	ResponseBody *PutCrmEventAttendeeResponseBody
}
